distributed/worker_memory.py

```diff
@@ -417,12 +417,14 @@ def memory_monitor(self, nanny: Nanny) -> None:
             return
 
         if self._last_terminated_pid != process.pid:
-            nanny_logger.warning(
+            msg = (
                 f"Worker {nanny.worker_address} (pid={process.pid}) exceeded "
                 f"{self.memory_terminate_fraction * 100:.0f}% memory budget. "
-                "Restarting...",
+                f"Restarting..."
             )
+            nanny_logger.warning(msg)
             self._last_terminated_pid = process.pid
+            nanny.log_event("worker-restart-memory", msg)
```

Comment:
The full text message is only relevant for the logger but the `log_event` stuff is supposed to be machine readable and prose will not help us much.

```suggestion
            msg = {
                "worker": nanny.worker_address,
                 "pid": process.pid,
                 "rss": memory,
            }
            nanny.log_event("worker-restart-memory", msg)
```

This is easier to parse automatically and contains the same / more information that the long string.

Reply:
Thx for the explanation, sounds good to me